### ISSUE TITLE
FPS calculation

### ISSUE THREAD

**Issue by Xemnas0** [ikostrikov/pytorch-a2c-ppo-acktr-gail#238]
While training, the number of frames used so far is computer as 
`total_num_steps = (j + 1) * args.num_processes * args.num_steps`

Shouldn't this be multiplied by the number of stacked frames (default 4)?

**Comment by ikostrikov** [comment]
For reporting result I'm using Monitor wrapper from OpenAI:
https://github.com/ikostrikov/pytorch-a2c-ppo-acktr-gail/blob/master/a2c_ppo_acktr/envs.py#L51

It properly computes the number of original environment frames.

**Comment by Xemnas0** [comment]
True, but I believe this is not taken into account in the print info while training, is it?
FPS therefore should stand for number of environment steps per second, and not frames per second.

**Comment by ikostrikov** [comment]
Yes, that's true. After the NeurIPS deadline I will change it to "env. steps per second" to avoid any potential confusion.
